Automattic/jetpack
# Hoist the Tracking package into Connection.

This change continues the hoisting project and hoists the Tracking package into Connection. No functional changes in this PR.

#### Changes proposed in this Pull Request:

* Moves code to the Connection package.
* Moves test suites to the Connection package.
* Removes the Tracking dependency from every composer.json file.
* Marks the Tracking page as abandoned. 

#### Jetpack product discussion


* p9dueE-4Ep-p2

#### Does this pull request change what data or activity we track or use?


* No

#### Testing instructions:





* Test the Tracking code using its PHPUnit test suite.
* Test Tracking by going to the dashboard. The client side part of it is easy to test using the debug switch:
* Enter this totally safe code into your JavaScript console: `localStorage.setItem('debug', 'dops:analytics');`
* This will turn on tracking debugging, you should see events being logged. Make sure you have tracking protection disabled in your browser.
